scripts/push-manifest.sh

```diff
@@ -62,24 +56,4 @@ OUT=$(manifest-tool $MT_ARGS push from-args \
                     --template "$TARGET"-ARCH \
                     --target "$TARGET")
 
-echo "$OUT"
-if [ -z "$TRUST" ]; then
-    echo "Not signing $TARGET"
-    exit 0
-fi
-
-# Extract sha256 and length from the manifest-tool output
```

Comment:
I think you can remove all the manifest-tool parts above as it is only used here

Reply:
we can probably remove this script. it;'s only used for kernel build and the alpine base. if we can replace the manifest tool with docker than there is no need for this script

Reply:
I'll keep it here. We can remove it in a follow-up PR that removes manifest-tool and related bits